pickee_frontend/src/components/TestComponent.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
I know this is a test component, but let's be cautious with having `template` | `script` |  `style` in the same `Vue` file for **this project**.

Reply:
You mean to avoid “single-file components” and instead use something like this?
`<!-- my-component.vue -->
<template>
  <div>This will be pre-compiled</div>
</template>
<script src="./my-component.js"></script>
<style src="./my-component.css"></style> `

Reply:
yep

Reply:
Ok, we can do that